Singer.API/ClientApp/src/app/modules/careusers/components/care-user-details/care-user-details.component.html

```diff
@@ -205,6 +248,16 @@ <h3 class="section-title">Zorg gegevens</h3>
 
       <!-- Button Row -->
       <mat-dialog-actions class="button-row">
+         <!-- Close Button -->
+         <button
+            mat-raised-button
+            (click)="closeForm()"
+            title="Formulier sluiten"
+            aria-label="Formulier sluiten"
+         >
+            Sluiten
+         </button>
+
          <!-- Add Button -->
          <button
```

Comment:
You removed the `color="accent"` from the close button, which is good, but I think it should be added to the save button. @BerendWouters  what do you think?

Reply:
The team decided against this.